sources/packages/backend/apps/api/src/route-controllers/confirmation-of-enrollment/_tests_/e2e/confirmation-of-enrollment.institutions.confirmEnrollment.e2e-spec.ts

```diff
@@ -89,6 +90,41 @@ describe("ConfirmationOfEnrollmentInstitutionsController(e2e)-confirmEnrollment"
     );
   });
 
+  it("Should allow the COE confirmation when the application is on Enrolment status and all the conditions are fulfilled for Part Time offering.", async () => {
+    // Arrange
+    const application = await saveFakeApplicationDisbursements(
+      appDataSource,
+      {
+        institution: collegeC,
+        institutionLocation: collegeCLocation,
+      },
+      {
+        applicationStatus: ApplicationStatus.Enrolment,
+        offeringIntensity: OfferingIntensity.partTime,
+      },
+    );
+    const [firstDisbursementSchedule] =
+      application.currentAssessment.disbursementSchedules;
+    const endpoint = `/institutions/location/${collegeCLocation.id}/confirmation-of-enrollment/disbursement-schedule/${firstDisbursementSchedule.id}/confirm`;
+    // Act/Assert
+    await request(app.getHttpServer())
+      .patch(endpoint)
+      .send({ tuitionRemittanceAmount: 1 })
+      .auth(
+        await getInstitutionToken(InstitutionTokenTypes.CollegeCUser),
```

Comment:
Even being copied from the above E2E, the token is preferred to be acquired at the end of the `Arrange` section.

Reply:
I am not able to see this change reflected in the PR. Is there some commit missing?

Reply:
I believe there was a misunderstanding, the comment was about the token to have it aligned with other E2E tests.
Please see the one below from a sibling file.
![image](https://github.com/user-attachments/assets/44cd3f61-1837-4c5e-aa4c-7a6fcf0cd781)